devito/operator/operator.py

```diff
@@ -285,6 +290,11 @@ def _lower(cls, expressions, **kwargs):
     def _rcompile_wrapper(cls, **kwargs0):
         raise NotImplementedError
 
+    @classmethod
+    def _load_dtype_mappings(cls, **kwargs):
+        lang: type[LangBB] = cls._Target.DataManager.lang
+        ctypes_vector_mapper.update(lang.mapper.get('types', {}))
```

Comment:
Now that's a bit tricky because this updates a global `ctypes_vector_mapper` which might lead to odd behavior building multiple operators with different languages.
Do you know where it's called and needs those `types` ? I.e can the mapper be "local" to the operator and passed there?